### PR TITLE
add 1715 labs subject images

### PR DESCRIPTION
**Pull request by camallen**
Add new subject images to the dev classifier. This was a request from Dan to have more relevant testing images for 1715 labs when showing off the UI capabilities.

default behaviour: https://pr-5296.pfe-preview.zooniverse.org/dev/classifier
1715 labs behaviour: https://pr-5296.pfe-preview.zooniverse.org/dev/classifier?1715labs=true

Staging branch URL: 

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/pages/dev-classifier/mock-data.coffee [hunk 1]
```diff
@@ -518,20 +518,33 @@ workflow = apiClient.type('workflows').create
...
-      {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/very-tall.jpeg"} # //lorempixel.com/1000/1900/animals/4
-      {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/small.jpeg"} # //lorempixel.com/400/300/animals/4
-      {'text/plain': "https://static.zooniverse.org/preview.zooniverse.org/panoptes-front-end/highlighter-task/assets/dev-classifier/algernon.txt"}
-    ]
+    if labs1715QueryPresent()
+      [
+        {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/1715_labs/london_street_view.jpeg"}
+        {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/1715_labs/greenwich_satellite_image.jpeg"}
+        {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/1715_labs/building_plans.jpeg"}
+      ]
```
Comment (eatyourgreens) [inline app/pages/dev-classifier/mock-data.coffee]: Without the `tex/plain` subject, the dummy text task might fail.

**Comment by eatyourgreens** [comment]
I'm inclined to say replace the dummy files with these new ones. Also, we should add some audio and video subjects to show that off. That would probably involve mocking the subject queue to include more than one subject, or adding a selector to change the mock subject.

File: app/pages/dev-classifier/mock-data.coffee (hunk 1 above)
Comment (camallen) [reply app/pages/dev-classifier/mock-data.coffee]: nice catch - thanks.

Comment (rogerhutchings) [reply app/pages/dev-classifier/mock-data.coffee]: It doesn't work, but it doesn't crash the classifier 🎉

**Comment by camallen** [comment]
> I'm inclined to say replace the dummy files with these new ones.

Happy to but doesn't everyone want to see that picture of the Rhino?

> Also, we should add some audio and video subjects to show that off. 

I don't have any examples of these...

> That would probably involve mocking the subject queue to include more than one subject, or adding a selector to change the mock subject.

Maybe in a latter PR, this was to get something quick up for 1715 labs...don't want to invest much time in it, ££

File: app/pages/dev-classifier/mock-data.coffee (hunk 1 above)
Comment (eatyourgreens) [reply app/pages/dev-classifier/mock-data.coffee]: 👍 I've been thinking the text task should automatically bring the text frame into view, for convenience. Maybe by adding a new static property for tasks, which is the default file type that those tasks act on.

**Pull request merged**